vendor_skins/Evercast/CI/configure-script-win.cmd

```diff
@@ -0,0 +1,15 @@
+rem ---------------------------------------------------------------------
+cmake ^
```

Comment:
I sure did not.  I did adjust it a little (to use a more recent version of VS) but did not commit that change, since that's a user preference, and not an upgrade.